lib/error/error_manager_req.cpp

```diff
@@ -50,11 +50,13 @@ void ErrorManagerReq::subscribe_all_errors(const ErrorCallback& callback, const
 
 void ErrorManagerReq::on_error_raised(const Error& error) {
     if (std::find(allowed_error_types.begin(), allowed_error_types.end(), error.type) == allowed_error_types.end()) {
-        throw EverestBaseLogicError("Error can't be raised by module_id " + error.origin.module_id +
-                                    " and implemenetation_id " + error.origin.implementation_id);
+        EVLOG_error << "Error can't be raised by module_id " << error.origin.module_id << " and implemenetation_id "
+                    << error.origin.implementation_id << ", ignored.";
+        return;
     }
```

Comment:
This check kind of duplicates the check on `ErrorManagerImpl`-side.
In case of passing the first check this means to be in a very corrupted state, which should crash/throw. Since a raised error is not accepted and not handled.